tests/accessor/accessor_common.h

```diff
@@ -864,6 +864,16 @@ decltype(auto) get_subscript_overload(const AccT& accessor, size_t index) {
   if constexpr (dims == 3) return accessor[index][index][index];
 }
 
+template <int dims>
+void add_id_linear(sycl::id<dims>& id, size_t size) {
+  for (int i = dims - 1; i >= 0; i--) {
+    if (i < size) {
```

Comment:
Doesn't compiler worry about signed/unsigned comparison?

Reply:
Changed size to int